src/Geosuggest.tsx

```diff
@@ -650,7 +650,7 @@ export default class extends React.Component<IProps, IState> {
     );
 
     return (
-      <div className={classes} id={this.props.id}>
+      <div className={classes} id={`${this.props.id}--geosuggest`}>
```

Comment:
Here its debatable what could be a good wrraper id..it just can't be the same as the one in the input.